src/main/java/com/thagorn/hex/hexserver/TestDataInitializer.java

```diff
@@ -22,7 +22,9 @@ public TestDataInitializer(LevelRepository repository) {
     public void run(String... args) throws Exception {
         Stream.of("Test Level 1", "DND level", "The other test level",
                 "Pretends to be the third test level").forEach(name ->
-                repository.save(new Level(name))
+                {if (repository.findByName(name) == null) {

```

Comment:
the more streamy way to do this is to add a .filter() in between .of() and .forEach()

Reply:
Guess I should figure out what's up with "Stream" then.